content/en/references/external-ports.md

```diff
@@ -3,18 +3,38 @@ title: "External Service Port Range"
 
-Another option is using the *external service port range*, which - e.g. in OpenSearch - is enabled by using the [`OPENSEARCH_ENDPOINT_STRATEGY=port`]({{< ref "opensearch#endpoints" >}})).  
-The external service port range is a set of pre-defined ports (by default `4510-4559`). LocalStack will chose a free port within this range when starting an external service.
-These ports need to be accessible from outside the Docker container and in turn allows to directly access an external service (as opposed to using LocalStack as a proxy).
+## Proxy Functionality for External Services
 
-The port range is configurable by using the environment variables `EXTERNAL_SERVICE_PORTS_START` and `EXTERNAL_SERVICE_PORTS_END`. This results in the external service port range `(EXTERNAL_SERVICE_PORTS_START, EXTERNAL_SERVICE_PORTS_END]` (i.e. the `EXTERNAL_SERVICE_PORTS_END` is _not_ included in the range).
+LocalStack offers a proxy functionality to access external services indirectly. In this approach, LocalStack assigns local domains to the external services based on the individual service's configuration. 
+
+For instance, if OpenSearch is configured to use the [`OPENSEARCH_ENDPOINT_STRATEGY=domain`]({{< ref "opensearch#endpoints" >}}) setting, a cluster can be reached using the domain name `<domain-name>.<region>.<engine-type>.localhost.localstack.cloud`. Incoming messages to these domains are relayed to servers running on ports that do not require external accessibility.
+
+## Direct Access with External Service Port Range
+
+An alternative approach to accessing external services is by utilizing the _external service port range_. This method, applicable to services like OpenSearch, is activated using the [`OPENSEARCH_ENDPOINT_STRATEGY=port`]({{< ref "opensearch#endpoints" >}}) configuration. The external service port range is pre-defined and set to `4510-4559` by default.
+
+When a LocalStack service starts an external service, it automatically selects an available port from within the specified range. The primary advantage of this approach is that these ports are accessible from outside the Docker container, allowing direct access to the external service without the need for LocalStack to act as a proxy.
+
+## Configuring the External Service Port Range
+
+To configure the external service port range, you can make use of the environment variables `EXTERNAL_SERVICE_PORTS_START` and `EXTERNAL_SERVICE_PORTS_END`. The range is defined as `(EXTERNAL_SERVICE_PORTS_START, EXTERNAL_SERVICE_PORTS_END]`, wherein the `EXTERNAL_SERVICE_PORTS_END` value is not included in the range.
+
+By adjusting these environment variables, you can customize the port range according to your requirements, granting you greater flexibility in managing external service access.
+
+## Running multiple LocalStack containers with Custom Port Mapping
```

Comment:
While it is great to have this in the documentation, it seems a bit hard to find.
When I want to start multiple localstack instances, I guess I wouldn't look for the docs on it in this page.
Maybe we should move this section to another page and reference the external service ports docs?

Reply:
We'll keep it here for now, maybe we find a better place for this documentation in another iteration.